Directory.Build.props

```diff
@@ -1,7 +1,7 @@
 <Project>
 
   <PropertyGroup>
-    <VersionPrefix>4.12.1</VersionPrefix>
+    <VersionPrefix>4.12.2-alpha</VersionPrefix>
```

Comment:
Best to avoid adding this here, lets just version it during the release flow.

Reply:
That's actually not part of this PR - it comes from the version-5.0.0 branch that we're targeting for this PR. I think you made the same comment on the net9.0 PR (which is also a PR targeting version-5.0.0).

Reply:
No idea how that happened. Reverted in any case.